cfme/containers/provider/openshift.py

```diff
@@ -76,12 +82,20 @@ def href(self):
 
     @property
     def view_value_mapping(self):
-        return {
+
+        mapping = {
             'name': self.name,
-            'prov_type': 'OpenShift Container Platform',
             'zone': self.zone,
+            'metrics_type': self.metrics_type
         }
 
+        if appliance_is_downstream():
```

Comment:
**Required**
This is using store.current_appliance, instead reference `self.appliance.is_downstream` to specifically query the appliance associated with this provider.

Reply:
:+1: